napari/_qt/qt_viewer.py

```diff
@@ -445,9 +446,14 @@ def _on_cursor(self, event):
         event : napari.utils.event.Event
             The napari event that triggered this method.
         """
-        cursor = self.viewer.cursor
+        cursor = self.viewer.cursor.style
+        # Scale size by zoom if needed
+        if self.viewer.cursor.scaled:
+            size = self.viewer.cursor.size * self.viewer.camera.zoom
+        else:
+            size = self.viewer.cursor.size
```

Comment:
whyyyy

Reply:
this is for painting in the labels layer to adjust the size of the paintbrush with zoom

Reply:
Ok. Problem is harder. Zoom is wrong...